### ISSUE TITLE
Wrong project version in 2.4.0 release

### ISSUE THREAD

**Issue by GiulioRomualdi** [marzer/tomlplusplus#110]
I noticed that in the [release 2.4.0](https://github.com/marzer/tomlplusplus/tree/v2.4.0) the cmake project version is 2.3.1 

https://github.com/marzer/tomlplusplus/blob/1baad21734d6d28a4efeb9ee3288c63487519bdb/CMakeLists.txt#L3

This is actually a problem when cmake `find_package` method is used in a consumer library.

Indeed 
```cmake
find_package(tomlplusplus 2.4.0 REQUIRED)
```
will fail with an error similar to this one
```
  Could not find a configuration file for package "tomlplusplus" that is
  compatible with requested version "2.4.0".

  The following configuration files were considered but not accepted:

    /usr/share/miniconda/envs/test/lib/cmake/tomlplusplus/tomlplusplusConfig.cmake, version: 2.3.1
```

On the latest master https://github.com/marzer/tomlplusplus/commit/4f21332bdd7555bbf9add1aafe82909a2f8aa52b the version is correct

**Comment by marzer** [comment]
Ah, thanks for reporting this @GiulioRomualdi. There have been a few other minor useful improvements since 2.4.0 too so perhaps it's time for me to publish a new release.

**Comment by marzer** [comment]
Released in [v2.5.0](https://github.com/marzer/tomlplusplus/releases/tag/v2.5.0) :)

**Comment by traversaro** [comment]
Thanks @marzer !
